sources/packages/backend/apps/db-migrations/src/sql/Applications/Drop-add-application-versioning-columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE
```

Comment:
Please follow the pattern of `Rollback-{migration_file_name}` pattern for the rollback file name.

`Rollback-add-application-versioning-columns.sql`